source/core.py

```diff
@@ -81,6 +81,8 @@ def restart(disableAddons=False):
 		pass
 	if disableAddons:
 		options.append('--disable-addons')
+	if debugLogging:
+		options.append('--debug-logging')

```

Comment:
Consider the following scenario:

1. User chooses to restart with debug logging.
2. User then restarts again normally (no debug logging).

In step 2, sys.argv will contain `--debug-logging`, but it isn't removed. Unless I'm missing something, that means debug logging will still be enabled on the second (normal) restart.

In contrast, --disable-addons is explicitly removed from sys.argv just before this, which prevents it from being propagated into a normal restart.